README.md

```diff
@@ -202,35 +195,16 @@ The Realm Dart package is `realm_dart`
     dart run realm_dart install
     ``` 
 
-* Enable generation of Realm schema objects.
-
-    * Add `build_runner` package to `dev_dependencies`.
-
-        ```
-        dart pub add build_runner --dev
-        ```
-    * Enable realm_generator by adding a `build.yaml` file to the application.
```

Comment:
Should we mention that the processed files can be configured with build.yaml?

Reply:
I think we could do that in the docs.